roles/reproducer/tasks/configure_controller.yml

```diff
@@ -468,3 +468,11 @@
       register: _sync_dep_install_result
       until: _sync_dep_install_result.finished
       retries: 20
+
+    - name: Configure the NTP service
+      vars:
+        timesync_dhcp_ntp_servers: true
+        timesync_ntp_servers:
+          - hostname: "{{ cifmw_ntp_server | default('clock.redhat.com') }}"
```

Comment:
the default value should be a public server

Reply:
done